README.md

```diff
@@ -16,11 +16,13 @@ Official mirrors:
 
 Compatibility tool will work on any modern 64-bit Linux distribution.
 All packaged games are compiled for Steam Runtime environment and require no
-additional dependencies. **Native Runtime is not supported.**
+additional dependencies.
+
+**Using Luxtorpeda with [Steam native runtime](https://wiki.archlinux.org/index.php/Steam/Troubleshooting#Steam_native_runtime) may or may not work, but is generally not supported.**
```

Comment:
I really prefer more specific wording in here.

I am not going to make changes/fix issues resulting from usage of native runtime. This is not "generally not supported" (as in "I might fix something occassionally") it is "not supported, I have no means of supporting it". Usage of native runtime is not recommended anyway…

Reply:
Hm, wanted to indicate that "it works", but maybe I'll choose something else. I have to update this PR anyway to match the boxtron one